torch/onnx/errors.py

```diff
@@ -42,3 +50,47 @@ def __init__(
                 "it with the right domain and version."
             )
         super().__init__(msg)
+
+
+class SymbolicValueError(OnnxExporterError):
+    """Errors around TorchScript values and nodes."""
+
+    def __init__(self, msg: str, value: _C.Value):
+        message = (
+            f"{msg}  [Caused by the value '{value}' (type '{value.type()}') in the "
+            f"TorchScript graph. The containing node has kind '{value.node().kind()}'.] "
+            f"\n    (node defined in {value.node().sourceRange() or 'unknown source code location'})"
```

Comment:
If the message actually showing `'unknown source code location'` is not very useful and redundant. Possibly making this nothing or specify the location?

Reply:
Done